### DISCUSSION TITLE
Lesson 4: Ethereum chainlink

### DISCUSSION THREAD

**Discussion by igboBoydev** [Cyfrin/foundry-full-course-cu#1403] [Q&A]
Hello everyone,

Please I am unable to get the sample code snippet on chainlink's data feeds for converting ETH/USD as shown in the Lesson for of the course pricely at 04:29:36 hours of the course.

Please any help on how I can get the code snippet for ETH/USD ?

What is currently on the Chainlink data feeds website is for BTC/USD.

**Comment by smart-bonds** [comment]
You can use the code snippet as provided on Chainlink's site, just change the contract address to 0x694AA1769357215DE4FAC081bf1f309aDC325306 (ETH/USD).

The complete list of price feed contract addresses on Sepolia can be found [here](https://docs.chain.link/data-feeds/price-feeds/addresses?network=ethereum&page=1#sepolia-testnet).
